pubspec.yaml

```diff
@@ -8,23 +8,23 @@ environment:
   sdk: ">3.0.0"
 
 dependencies:
-  android_id: ^0.1.3+1
-  connectivity_plus: ^3.0.3
-  device_info_plus: ^8.0.0
+  android_id: ^0.3.6
+  connectivity_plus: ^5.0.2
+  device_info_plus: ^9.1.1
   equatable: ^2.0.5
   flutter:
     sdk: flutter
-  flutter_webrtc: ^0.9.45
-  http: ^0.13.5
-  intl: ^0.18.1
+  flutter_webrtc: ^0.9.47
+  http: ^0.13.6
```

Comment:
There's a newer version of `http` (`1.1.2`), but explorer-ui depends on `gallery_saver` `2.3.2`, which depends on `http` `^0.13.3`.